### ISSUE TITLE
v1.4: assertionerror in untip

### ISSUE THREAD

**Issue by ptrebert** [marbl/verkko#163]
Hi,

I have two v1.4 runs with the following assertion failing:

```
>>> file: untip.err
>>> untip error
Find lengths
Fix coverage
Traceback (most recent call last):
File "[...]/lib/verkko/scripts/get_original_coverage.py", line 91, in <module>
assert parts[0] not in mapping
AssertionError
```

I can share the Verkko run directory for debugging purposes, but only via Globus (push or pull).

Best,
Peter

**Comment by skoren** [comment]
I think this is likely a similar cause as #165. Can you share the assemblies via globus, we only need the 2,4,5,6 folders, you can skip the rest to save space.

**Comment by ptrebert** [comment]
Thanks, I triggered the data transfer for both samples (subfolder 6- does not yet exist, though). You'll find the data under "/verkko-debug/ ..."

**Comment by skoren** [comment]
This should be fixed by the recent commit. You should be able to replace the changed script in your conda install just to confirm. Once we fix your other issues we can make a 1.4.1 bug fix release. Please upload your other failed runs so we can debug those as well.

**Comment by ptrebert** [comment]
Thanks, I will start the Globus transfer for the other buggy runs

**Comment by ptrebert** [comment]
confirmed fixed via 3119b39

Thanks
